tests/ui/exact-harness/incomplete-harness-name/expected

```diff
@@ -0,0 +1 @@
+error: no harnesses matched the harness filter: `check_blah`
```

Comment:
nit: can you change this message for the case where `--exact` is provided? something like:
```
error: no harnesses matched the name: `check_blah`
```